modules/git/repo_tree_nogogit.go

```diff
@@ -30,7 +30,7 @@ func (repo *Repository) getTree(id SHA1) (*Tree, error) {
 		if err != nil {
 			return nil, err
 		}
-		tag, err := parseTagData(data)
+		tag, err := parseTagData(repo, data)
 		if err != nil {
 			return nil, err
 		}
```

Comment:
the other optin would be to inject repo into tag here - but current pull reduce error surface and dedup code